lib/gooddata/commands/auth.rb

```diff
@@ -22,7 +22,7 @@ class << self
         #   * :environment [String] Environment - DEVELOPMENT, TEST, PRODUCTION
         #   * :server => [String] Server - https://secure.gooddata.com
         def ask_for_credentials(credentials_file_path = Helpers::AuthHelper.credentials_file)
-          puts 'Enter your GoodData credentials.'
+          GoodData.logger.info('Enter your GoodData credentials.')
```

Comment:
Do we really want to redirect communication with the user on command line to the logger?

Reply:
I've them back.